iceoryx2-cxx/tests/src/service_publish_subscribe_tests.cpp

```diff
@@ -494,7 +494,7 @@ TYPED_TEST(ServicePublishSubscribeTest, write_from_fn_send_receive_works) {
 
     auto iterations = 0;
     for (const auto& item : recv_sample.payload()) {
-        ASSERT_THAT(item.a, Eq(DummyData::DEFAULT_VALUE_A + iterations));
+        ASSERT_THAT(item.a, Eq(DummyData::DEFAULT_VALUE_A + static_cast<uint64_t>(iterations)));
```

Comment:
I think we can get rid of the cast wenn `iterations` will be set to `uint64_t` further above.